src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/RadiobuttonAccessibility.h

```diff
@@ -28,5 +28,6 @@
 @interface RadiobuttonAccessibility : ButtonAccessibility <NSAccessibilityRadioButton> {
 
 };
-- (id)accessibilityValue;
+- (NSAccessibilityRole _Nonnull)accessibilityRole;
+- (id _Nonnull)accessibilityValue;
```

Comment:
Same as earlier. Can't this be null?

Reply:
Same here, value represents the state of the control and can not be set to null.